crypto/BRCryptoNetwork.c

```diff
@@ -509,7 +514,10 @@ cryptoNetworkCreateAddressFromString (BRCryptoNetwork network,
             return cryptoAddressCreateFromStringAsETH (string);
 
         case BLOCK_CHAIN_TYPE_GEN:
-            return cryptoAddressCreateFromStringAsGEN (string);
+        {
+            BRGenericAddress genericAddress = gwmNetworkCreateAddress(network->u.gen.net, string);
+            return cryptoAddressCreateAsGEN(NULL, genericAddress);
```

Comment:
This is the one issue remaining.  We have no way to get the generic wallet manager from here.  So either we find a way or we ensure that BRCryptoAddress can operate without one.

Reply:
See my comment below on `BRGenericNetworkRecord`.